### ISSUE TITLE
[Accessibility][Screen Reader] [fd-form-message] Screen reader cannot get the messages

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13455]
### Describe your proposal

Screen reader should read the fd-form-message content.

<img width="1342" height="708" alt="Image" src="https://github.com/user-attachments/assets/8ef493af-b5d6-4b73-9d8f-d75399115e52" />

### Which versions of Angular and Fundamental Library for Angular are affected?

lastest version

### Steps to Reproduce (for bugs)

1. Open form message example and find 【**Form Message State in Input Type**】 [https://fundamental-ngx.netlify.app/#/core/form-message](https://fundamental-ngx.netlify.app/#/core/form-message)
2. Open screen reader
3. Focus on the input ,the screen reader reads wrong content

<img width="1342" height="708" alt="Image" src="https://github.com/user-attachments/assets/9e89d813-2d92-413a-9a9f-730735d52814" />

### Declaration

- [x] I'm not disclosing any internal or sensitive information.
